src/Files.App/Actions/Content/Background/SetAsLockscreenBackgroundAction.cs

```diff
@@ -18,20 +20,21 @@ public override RichGlyph Glyph
 
 		public override bool IsExecutable =>
 			base.IsExecutable &&
-			context.SelectedItem is not null;
+			ContentPageContext.SelectedItem is not null;
 
 		public override Task ExecuteAsync(object? parameter = null)
 		{
-			if (context.SelectedItem is null)
+			if (ContentPageContext.SelectedItem is null)
 				return Task.CompletedTask;
 
 			try
 			{
-				return WindowsWallpaperService.SetLockScreenWallpaper(context.SelectedItem.ItemPath);
+				return WindowsWallpaperService.SetLockScreenWallpaper(ContentPageContext.SelectedItem.ItemPath);
 			}
 			catch (Exception ex)
 			{
 				ShowErrorDialog(ex.Message);
+				App.Logger.LogWarning(ex, ex.Message);
```

Comment:
I don't think logging helps us because there isn't anything to do for this type of exception. The best thing is to notify the user and prevent a crash.